src/com/lilithsthrone/game/character/GameCharacter.java

```diff
@@ -2479,10 +2571,12 @@ public String incrementObedience(float increment, boolean applyJobPerkGains) {
 		
 		this.obedience = Math.max(-100, Math.min(100, obedience+increment));
 		
+		float shownObedience = Math.round(this.obedience);
+		
 		return UtilText.parse(this,
```

Comment:
add an if statement for isPlayer and write two versions of the same string, one for the npc, one for the player. It's hard to read text with a ton of ternaries in there.

Reply:
Yeah that was pretty messy, fixed it